e107_languages/English/admin/lan_theme.php

```diff
@@ -130,4 +130,5 @@
 define("TPVLAN_93", "Selection");
 define("TPVLAN_94", "Site theme changed to [x].");
 define("TPVLAN_95", "Skin");
+define("TPVLAN_96", "Developer/Debug Mode: Scanning theme images folder for new media to import.");
 
```

Comment:
Thanks, but we shouldn't translate debug or developer information.

Reply:
Ok